web/src/hotkey.js

```diff
@@ -792,6 +768,18 @@ export function process_hotkey(e, hotkey) {
         return true;
     }
 
+    // Handle hotkeys for active popovers here which can handle keys other than `menu_dropdown_hotkeys`.
+    // We force user to close the popover if they want to use the hotkeys for other actions. This is to
+    // avoid confusion for the user since a hotkey can handled in one popover but not in others resulting
+    // in inconsistent behavior of some hotkeys working and some not.
+    if (navbar_menus.is_navbar_menus_displayed()) {
+        return navbar_menus.handle_keyboard_events(event_name);
+    }
+
+    if (popover_menus.is_message_actions_popover_displayed()) {
+        return message_actions_popover.handle_keyboard_events(event_name);
+    }
```

Comment:
Where did this come from?

Reply:
message_actions_popover also handles other than `menu_dropdown_hotkeys`, which is `i`, so it is necessary to handle it here too.

Reply:
I think a better approach would be let user open compose / search while these popovers are active. Modified the commit to do so and added a commit to hide popovers when starting search, compose etc.